src/cudadev/CUDADataFormats/TrackingRecHit2DSOAView.h

```diff
@@ -14,11 +14,9 @@ namespace pixelCPEforGPU {
 
 class TrackingRecHit2DSOAView {
 public:
-  static constexpr uint32_t maxHits() { return gpuClustering::MaxNumClusters; }
-  using hindex_type = uint16_t;  // if above is <=2^16
+  using hindex_type = uint32_t;  // if above is <=2^32
```

Comment:
this is no esthetic

Reply:
try to revert this

Reply:
if I change this I get a failed assert:
```
/data/user/fwyzard/pixeltrack-standalone/src/cudadev/CUDADataFormats/TrackingRecHit2DHeterogeneous.h:111:62: error: static assertion failed
   static_assert(sizeof(TrackingRecHit2DSOAView::hindex_type) == sizeof(float));
```

Reply:
ok. the storing area changed as well. so not so trivial

Reply:
maybe there is a not so hard way to revert the full PR....

Reply:
not obvious, was done BEFORE the merge (or at least not by me in CMSSW master)

Reply:
Yes, I think it was done in the Patatrack branch - before the merge in the upstream CMSSW and the following clean up.

Reply:
Here: https://github.com/cms-patatrack/cmssw/pull/583